Apps/WebClient/src/ClientApp/app/components/timeline/comment.vue

```diff
@@ -19,13 +54,48 @@
 <script lang="ts">
 import Vue from "vue";
 import UserComment from "@/models/userComment";
-import { Prop, Component } from "vue-property-decorator";
+import { Prop, Component, Emit, Watch } from "vue-property-decorator";
+import { faEllipsisV, IconDefinition } from "@fortawesome/free-solid-svg-icons";
 
 @Component
 export default class CommentComponent extends Vue {
-  @Prop({ default: "Default" }) comment!: UserComment;
+  @Prop() comment!: UserComment;
+  @Prop() editing!: UserComment;
```

Comment:
These two variables look kind of odd. From the looks of it this component represents a single comment but has two internal members representing it?

Reply:
comment populates the comment component with information while editing refers to the comment that is currently being edited for comparison